src/apiCalls.js

```diff
@@ -0,0 +1,4 @@
+export const getAllMovies = () => {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
Doing our fetching in this file is a much more SRP way of getting our data. Looks good.